frontend/src/components/auth/auth.tsx

```diff
@@ -29,6 +36,12 @@ const Auth: FC = () => {
     dispatch(authActions.signUp(payload));
   };
 
+  useEffect(() => {
+    if (dataStatus === DataStatus.FULFILLED) {
+      navigate(AppRoute.ROOT);
+    }
+  }, [dataStatus]);
```

Comment:
```suggestion
```
just change this logic

if (hasUser) {
  return <Redirect /> 
}
<img width="282" alt="image" src="https://user-images.githubusercontent.com/42296182/183714155-e2bf76e9-fb5f-45dc-a781-7b9c6d7b2b73.png">